docs/mdx-plugins/remark-export-headings.js

```diff
@@ -1,8 +1,8 @@
 const visit = require('unist-util-visit');
 
 /**
- * @typedef {import('mdast').Root} Root - https://github.com/syntax-tree/mdast#root
- * @typedef {import('mdast').Heading} Heading - https://github.com/syntax-tree/mdast#heading
+ * @typedef {import('@types/mdast').Root} Root - https://github.com/syntax-tree/mdast#root
```

Comment:
Technically, importing `mdast` is wrong because there is no `mdast` package. TypeScript/JS IntelliSense does handle it correctly, because it automatically prepends `@types/`. But we should be explicit here.